cleanupsdemo/cleanups_demo.go

```diff
@@ -22,14 +24,24 @@ func main() {
 	// of the thunks that are registered with `runtime.SetFinalizer()` in
 	// group.go and collect.go, which close channels and call cancel functions.
 
-	println("leaking ~", cycles*(batchSize+2), "goroutines from abandoned group executors...")
+	println("leaking goroutines from group executors...")
+
+	// Dependent contexts are always canceled, too
+	leakDependent := func(ctx context.Context) {
+		ctx, cancel := context.WithCancel(ctx)
+		_ = cancel
```

Comment:
what's the purpose of this assignment?

Reply:
it yells at me if i write `ctx, _ = withcancel` and it yells at me even more if i don't do anything with `cancel`. this is just being explicit that we are discarding the cancel here, which is fine